pkgs/development/python-modules/scipy/default.nix

```diff
@@ -78,10 +78,10 @@ in buildPythonPackage {
   # Relax deps a bit
   postPatch = ''
     substituteInPlace pyproject.toml \
-      --replace 'meson-python>=0.12.1,<0.15.0' 'meson-python' \
-      --replace 'numpy==' 'numpy>=' \
-      --replace "pybind11>=2.10.4,<2.11.1" "pybind11>=2.10.4,<2.12.0" \
-      --replace 'wheel<0.41.0' 'wheel'
```

Comment:
Nice! I'll pull this change into https://github.com/NixOS/nixpkgs/pull/282506 . Could you remove it from here please?

Reply:
Done, and #282506 should be retargeted to -next then.

Reply:
> Done, and #282506 should be retargeted to -next then.

Done.